ADR/ADR038-add-solidqueue-to-forms-runner.md

```diff
@@ -0,0 +1,46 @@
+# ADR038: SolidQueue and ActiveJob for managing background processing
+
+Date: 2024-11-25
+
+## Status
+
+Approved
+
+## Context
+
+Forms will be using AWS SES to deliver emails to form processors with files attached to the emails.
+
+For our team to successfully process submission emails we must have:
+* a way to retry sending the email on failed submissions
+* a queuing mechanism to send emails asynchronously
+* a way to retry the email on failed submissions, for example if an email bounces.
```

Comment:
How is this different to "a way to retry sending the email on failed submissions" on line 14?

Reply:
Ah good spot - this was a copy paste combination of things two of us wrote